docs/how-to-guides.md

```diff
@@ -32,6 +32,199 @@ Everything about test data generation and its adjustable parameters can
 be found in the [Generating a Test Dataset with niceML](
 generate-data.md#step-3-customizing-data-generation-optional)
 
+## How to write my own pipeline
+NiceML provides configurable, re-usable machine learning 
+pipelines. NiceML comes with pre-configured pipelines for object detection, 
+semantic semgmentation, regression, classification and others. But what if 
+your problem is not covered (yet) by the pipeline portfolio niceML offers?
+This guide will walk you through how to set up your first custom pipeline.
+The goal of the pipeline is to orchestrate a classifier that can distinguish
+between daisies and roses.
+
+1. Download the images from [Kaggle](https://www.kaggle.com/datasets/alxmamaev/flowers-recognition)
+and store the roses and daisies images in a `data` folder in your niceML project.
+2. Let us start configuring at the highest level of niceML: The job 
+configuration. For this, navigate to `configs/jobs/job_train/job_train_cls` 
+and create a new file `job_train_cls_binary_flowers.yaml`.
+3. Paste the content from `job_train_cls_binary.yaml` which is a good
+starting point for our own pipeline.
+4. Adjust job parameters in `job_train_cls_binary_flowers.yaml`. Many parameters are 
+pre-configured and do not require adjustment. However, we want to configure
+our own **training**, **prediction**, **experiment_id** and
+**input data location**. In the job configuration, we can already point
+to the location where the Kaggle images are stored and to the dagster training and prediction
+operation configurations we are going to write in the next step.
+   ```yaml
+   # train binary classification
+   defaults:
+     # experiment
+     - ops/experiment@ops.experiment.config: op_experiment_default.yaml
+     # train
+     - /ops/train@ops.train.config: op_train_cls_binary_flowers.yaml
+     # prediction
+     - /ops/prediction@ops.prediction.config: op_prediction_cls_binary_flowers.yaml
+     # analysis
+     - /ops/analysis@ops.analysis.config: op_analysis_cls_binary_flowers.yaml
+     # experiment tests
+     - /ops/exptests@ops.exptests.config.tests: exptests_default.yaml
+     # experiment locations
+     - shared/locations@globals: exp_locations.yaml
+     - _self_
+   
+   hydra:
+     searchpath:
+       - file://configs
+   
+   globals:
+     exp_name: FlowersClsBinary
+     exp_prefix: FLOWERSCLB
+     data_location:
+       uri: ${oc.env:DATA_URI,./data}/kaggle_flowers
+   ```
+
+5. Configure training operation
+   1. We navigate to `configs/ops/train` and create `op_train_cls_binary_flowers.yaml`.
```

Comment:
Please replace the 'we' statements by simple imperatives to remain consistent with the existing documentation.

Reply:
This only applies strictly to explicit tasks to do by the user, not to informative/introducing documentation.